app/models.py

```diff
@@ -14,10 +17,55 @@
 
 from .custom_exceptions import AccessError
 from .json_util import to_json
+from .env import load_env
 
 session_context_id = contextvars.ContextVar('request_id', default=None)
 DBSession = scoped_session(sessionmaker(), scopefunc=session_context_id.get)
 
+env, cfg = load_env()
+strict = cfg['security']['strict']
+use_webhook = cfg['security']['slack']['enabled']
+webhook_url = cfg['security']['slack']['url']
```

Comment:
```suggestion
strict = cfg['security.strict']
use_webhook = cfg['security.slack.enabled']
webhook_url = cfg['security.slack.url']
```
To avoid errors if `cfg["security"]` or `cfg['security']['slack']` is `None`